js/jquery.mapael.js

```diff
@@ -290,9 +290,16 @@
                             delete plots[deletedPlots[i]];
                         }
                     }
+                // Delete ALL plots if deletedPlots is set to "all"
+                } else if (deletedPlots === "all") {
+                    $.each(plots, function(id, elem) {
+                        fnRemoveElement(elem);
+                    });
+                    // Empty plots array
+                    plots.length = 0;
```

Comment:
Good question. I actually did some research because I wasn't sure. Turned out `= {}`is not the best way to do it: it will keep any referenced array as is. See this SO answer: http://stackoverflow.com/a/1232046